ArduCopter/mode_guided.cpp

```diff
@@ -618,10 +618,7 @@ void ModeGuided::set_angle(const Quaternion &q, float climb_rate_cms_or_thrust,
     guided_angle_state.update_time_ms = millis();
 
     // log target
-    copter.Log_Write_GuidedTarget(guided_mode,
-                           Vector3f(guided_angle_state.roll_cd, guided_angle_state.pitch_cd, guided_angle_state.yaw_cd),
-                           false,
-                           Vector3f(0.0f, 0.0f, climb_rate_cms_or_thrust), Vector3f());
+    copter.Log_Write_Guided_Attitude_Target(guided_mode, guided_angle_state.roll_cd/100, guided_angle_state.pitch_cd/100, guided_angle_state.yaw_cd/100, 0, 0, 0, guided_angle_state.thrust, guided_angle_state.climb_rate_cms);
```

Comment:
Always prefer multiplication to division (where possible) division takes longer on the hardware.  We might as well do the same for climb_rate as well.

```suggestion
    copter.Log_Write_Guided_Attitude_Target(guided_mode, guided_angle_state.roll_cd * 0.01, guided_angle_state.pitch_cd * 0.01, guided_angle_state.yaw_cd * 0.01, 0, 0, 0, guided_angle_state.thrust, guided_angle_state.climb_rate_cms * 0.01);
```

Reply:
Done! Thanks